requirements.modules.txt

```
@@ -4,7 +4,7 @@ flask-sqlalchemy~=2.4.0
 marshmallow-sqlalchemy==0.22.3
 google-cloud-storage~=1.31.0
 google-cloud-pubsub==0.42.1
-cidc-schemas~=0.20.3
+cidc-schemas>=0.20.6
```

`~=a.b.c` == `>=a.b.c && <a.b+1.*`
so leave as it's been 
```suggestion
cidc-schemas~=0.20.6
```